src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -35,6 +35,12 @@ public class Exam extends DomainObject {
     @Column(name = "test_exam")
     private boolean testExam;
 
+    /**
+     * This boolean indicates weather monitoring is enabled
```

Comment:
```suggestion
     * This boolean indicates whether monitoring is enabled
```
Pretty sure you don't monitor rain 😉

Reply:
:D